Cargo.toml

```diff
@@ -10,8 +10,8 @@ license = "Apache-2.0 OR MIT"
 
 [dependencies]
 libc = ">=0.2.39"
-kvm-bindings = { version = ">=0.2.0", features = ["fam-wrappers"] }
-vmm-sys-util = ">=0.2.1"
+kvm-bindings = { version = ">=0.3.1", features = ["fam-wrappers"] }
```

Comment:
Should we just force the dependency on kvm-bindings as well to avoid problems in the future?